docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,220 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+
+
+# Managing neuroscience projects with **datashuttle**
+*Create, validate and transfer standardised project folders*
+
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-dark.png
+:align: center
+:class: only-dark
+:width: 650px
+```
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-light.png
+:align: center
+:class: only-light
+:width: 650px
+```
+<br>
+
+Maintaining a well-organised neuroscience project is hard. 
+
+Although everyone can appreciate the benefits of a tidy project
+folder, the practicalities of running an experiment often gets 
+in the way. Folder organisation 
+is low on the list of priorities when it
+comes to acquisition sessions spent managing complex systems 
+and experimental animals.
+
+However, the cost of small mistakes during data acquisition can be high.
+One misplaced character may mean sessions are missed by analysis 
+scripts or subject identifications duplicated.
+The best protection against such errors is automating the process
+through acquisition scripts—resulting in hours spent writing code 
+managing the naming and transfer of folders—a task entirely 
+unrelated to the central research goals.
+
+In our previous blog, we highlighted the benefits of data standardisation 
+for systems neuroscience, introducing the 
+[NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) 
+specification. 
+An immediate benefit of a widely-used standard is that the entire community
+can contribute to shared tools for project management.
+This means individual researchers don't waste time
+writing data-management code. 
+
+In this blog post we introduce 
```

Comment:
Earlier you say "In our previous blog", while here you say "In this blog post". I think it should be "blog post" in both cases.